### ISSUE TITLE
Bright FOG and camera light render issue 

### ISSUE THREAD

**Issue by Gweenteaneko** [ValveSoftware/Dota2-Gameplay#7203]
### Description

Theres a lighting problem of dimming if you move away your camera if view a
Away from the map or from your hero

**Comment by cephalopoid** [comment]
Same problem, GTS250

**Comment by Gweenteaneko** [comment]
> Same problem, GTS250

I've done a fresh install same problem

**Comment by expen6ve** [comment]
remove all launch options

**Comment by cephalopoid** [comment]
> 

It does not help...

**Comment by cephalopoid** [comment]
> remove all launch options

It does not help...

**Comment by Bloody3134** [comment]
> удалить все параметры запуска

It did not help

**Comment by shaileshgarg** [comment]
Can you post your Video Setting.

**Comment by Tomoraiki** [comment]
Will there be a fix for this problem?****

**Comment by shqshq321** [comment]
Same problem. Nothing helps, i'm a proud GT 330M user.

**Comment by Bloody3134** [comment]
Who knows how fixed this?

**Comment by Gweenteaneko** [comment]
i was able to review my replay and cant upload a video of dimming and im pretty sure its an render issue

and heres a **thicc muerta**
![20230308080752_1](https://user-images.githubusercontent.com/64141065/223587189-953cf008-6aed-4f25-af35-1adc0b9c5ced.jpg)
![20230308080854_1](https://user-images.githubusercontent.com/64141065/223587196-299e557a-d823-468f-8a0d-596a92d547c0.jpg)
![20230308080903_1](https://user-images.githubusercontent.com/64141065/223587201-7570a261-8ed0-468c-bffa-7a57fe13f8c1.jpg)

**Comment by Gweenteaneko** [comment]
> Can you post your Video Setting.

its nothing special I only use default and its a low spec computer

**Comment by elB3n1k** [comment]
Fog of War:

1st time playing in this patch and got the same issue like OP

**Comment by gangrelmal** [comment]
same here plus others, this path was disapointement, full of bugs, they reach the peak of absurd

**Comment by Jazzlook** [comment]
same promblem

**Comment by Jazzlook** [comment]
Same problem

**Comment by atsumia** [comment]
tried to fix it by reinstall drivers, start on vulkan, played with graphic options nothing helped. steel waiting to fix

**Comment by gangrelmal** [comment]
They made this mess, they gonna fix eventuly, I think...

**Comment by eclipsssex** [comment]
on gt9600 the fog of war is gone

**Comment by noir133** [comment]
valve fix it pls, i cant play without fog

**Comment by matheusbzevedo** [comment]
the same

**Comment by uniko23** [comment]
volvo fix pls

**Comment by volvofixIT** [comment]
I agree with the author, I have the same problem, I just want to play my favorite game, everything is fine with FPS and mmr I have 6k mmr I don't want to lose my progress

**Comment by matheusbzevedo** [comment]
I solved this problem adding "-vulkan" on launch options.

**Comment by sansetik** [comment]
OMG Valve, This Name delete bug and Hello New BUG --

**Comment by Gweenteaneko** [comment]
> OMG Valve, This Name delete bug and Hello New BUG --

Its not a bug its a feature says Valve

**Comment by sansetik** [comment]
![image](https://user-images.githubusercontent.com/55962156/223866481-ddee0bd5-c2cc-4dc3-b6bf-0025c89adf55.png)

**Comment by sansetik** [comment]
![image](https://user-images.githubusercontent.com/55962156/223866696-5f8bf400-8a3f-4809-8e46-c4c6fc744509.png)

**Comment by sansetik** [comment]
this feature ?

**Comment by Gweenteaneko** [comment]
> this feature ?
You're is square mine is also a blue circle

**Comment by x22five** [comment]
anyone fixed this issue?

**Comment by fufelshmerts** [comment]
> ![изображение](https://user-images.githubusercontent.com/55962156/223866696-5f8bf400-8a3f-4809-8e46-c4c6fc744509.png)

improve screen quality

**Comment by Gweenteaneko** [comment]
> anyone fixed this issue?

I try some code at lunch options the best is that picture above this

**Comment by x22five** [comment]
> > anyone fixed this issue?
> 
> I try some code at lunch options the best I got is that picture above this. And its unplayable

how long they need to fix this, or they dont core? i dont understand, this stupid new hero ruined my life f***

**Comment by x22five** [comment]
> I solved this problem adding "-vulkan" on launch options.

not for me :((((((((((((((((((((

**Comment by Gweenteaneko** [comment]
> > > anyone fixed this issue?
> > 
> > 
> > I try some code at lunch options the best I got is that picture above this. And its unplayable
> 
> how long they need to fix this, or they dont care? i dont understand, this stupid new hero ruined my life f***

I know the legit fix to this but it'll cost you buckaroos 
The main cause to this they remove support of opengl

**Comment by x22five** [comment]
> I know the legit fix to this but it'll cost you buckaroos The main cause to this they remove support of opengl

please be kind to update if you guys found a way to play this shit.

**Comment by elB3n1k** [comment]
I just finished a match, the fog of war issue is gone
but now another issue arises XD

#7371 Game Screen Render Quality <100% issue

**Comment by atsumia** [comment]
guys, did u fix fog of war?

**Comment by atsumia** [comment]
by update*

**Comment by sansetik** [comment]
I have very weak hardware, I have to set the screen processing quality to 100 then fps 0

**Comment by uniko23** [comment]
in launch options enter -dxlevel 101 should help

**Comment by shqshq321** [comment]
-dxlevel 101 in the launch options helped, but when the screen render quality is <100%, this bug appears. 
Generally playable, but during the game, sometimes the FPS drops a lot (GT 330M): 
![photo_2023-03-10_17-15-14](https://user-images.githubusercontent.com/127249580/224338014-f3f3a66d-3456-4f4c-a7d0-a9b4dd60c3da.jpg)

**Comment by gangrelmal** [comment]
-dxlevel 101 worked for me too but setting quality need to be in 100% so I lowered my screen resolution, is playable!

**Comment by shaileshgarg** [comment]
Seems like the issue is affecting only old GPUs, probably some graphics API update which the old GPUs don't support fully.

**Comment by yang7951** [comment]
Showing duplicate models causes FPS to drop, and if you can turn off the display effect may solve the problem, but I don't know how to turn it off
![3](https://user-images.githubusercontent.com/32860177/225024280-40525969-37b5-475b-9bbe-1f1c2167f232.png)

**Comment by Gweenteaneko** [comment]
> Showing duplicate models causes FPS to drop, and if you can turn off the display effect may solve the problem, but I don't know how to turn it off ![ed](https://user-images.githubusercontent.com/32860177/225032530-36f42759-7fe6-4abf-a61b-e6bb365e7cd2.jpg)

Try -dxlevel

**Comment by yang7951** [comment]
> > 显示重复模型会导致FPS下降，如果可以关闭显示效果可能会解决问题，但我不知道如何关闭它![艾德](https://user-images.githubusercontent.com/32860177/225032530-36f42759-7fe6-4abf-a61b-e6bb365e7cd2.jpg)
> 
> 尝试 -dxlevel

That's useless and doesn't improve FPS

**Comment by yang7951** [comment]
This method boosted my FPS, great[https://steamcommunity.com/app/570/discussions/0/3792632890312038053/](url)

**Comment by Gweenteaneko** [comment]
> 

How about the issue?

**Comment by yang7951** [comment]
Fog bug still exists, but it increases by 10FPS,I found a phenomenon that when the render quality is adjusted to <100, all models in the game will have a duplicate transparent model, including hero buildings, trees and so on. I think this is equivalent to running two games at the same time. When the hero uses stealth, the duplicate transparent model of the hero will disappear at the same time. I think if we can find the corresponding activity triggered by the stealth skill we should be able to filter out the transparent model. Some people say that this may be the Lod low precision compensation model error. If we can turn off Lod, I think we can solve the problem

**Comment by Gweenteaneko** [comment]
I already fix mine but I will do some more testing before posting

**Comment by brunogulart** [comment]
Now, to play DotA, you will need at least an Nvidia 600 series+ video card, an AMD 7000 series+ video card, an Intel HD 5000 or newer.

We also plan to reject all processors without SSE4.1 support.

The very appeal of the developers will be stuffed under the spoiler: https://i.imgur.com/cjkN9oi.png

that's basically it, the valve discontinued support for old boards that's why it's causing a problem, and it won't be solved, an upgrade will be necessary, I found out about it after researching a lot, I found this information on a Russian website, I don't know if it's true, but It seems to me that yes because the correction patches didn't fix it.

Font: 
https://dota2.ru/forum/threads/dota-2-prekratila-podderzhku-staryx-videokart.1529047/

**Comment by glarkmartius** [comment]
fog issue since muerta uodated.

**Comment by CSTGluigi** [comment]
trees beside dire and radiant base which can't block the view, as if they don't exist.
Actually it can block the view. It means that when you are in the Woods, you think there is no one outside the tree, because there is no fog of war, but there is someone.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/88568921/8ec19c15-efd7-441d-b3a8-a139c9c520cf)

**Comment by brunogulart** [comment]
Now, to play DotA, you will need at least an Nvidia 600 series+ video card, an AMD 7000 series+ video card, an Intel HD 5000 or newer.

We also plan to reject all processors without SSE4.1 support. within these settings?

**Comment by CSTGluigi** [comment]
> Now, to play DotA, you will need at least an Nvidia 600 series+ video card, an AMD 7000 series+ video card, an Intel HD 5000 or newer.
> 
> We also plan to reject all processors without SSE4.1 support. within these settings?
well, i found in real game or create room to test  then everything I see is normal. My bad

**Comment by brunogulart** [comment]
I don't have much patience and time to solve the problem, formatting your computer and reinstalling dota will probably solve your problem.

**Comment by Spaider15** [comment]
save for me, happened after i have installed Mac OS Sonoma
<img width="295" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/12617760/26216451-ca51-4feb-a3b9-e9d575b2592c">

**Comment by ToanHoangHuy** [comment]
> same for me, happened after i have installed Mac OS Sonoma <img alt="image" width="295" src="https://user-images.githubusercontent.com/12617760/271366149-26216451-ca51-4feb-a3b9-e9d575b2592c.png">

Yes, that's right. I encountered an issue after updating my macOS from Ventura to Sonoma. Please note that I was able to play Dota 2 normally on Ventura.

**Comment by Spaider15** [comment]
> > same for me, happened after i have installed Mac OS Sonoma <img alt="image" width="295" src="https://user-images.githubusercontent.com/12617760/271366149-26216451-ca51-4feb-a3b9-e9d575b2592c.png">
> 
> Yes, that's right. I encountered this issue after updated my macOS from Ventura to Sonoma. Please note that I was able to play Dota 2 normally on Ventura.

it works on windowed mode but you can't select full screen resolution

**Comment by ToanHoangHuy** [comment]
> > > same for me, happened after i have installed Mac OS Sonoma <img alt="image" width="295" src="https://user-images.githubusercontent.com/12617760/271366149-26216451-ca51-4feb-a3b9-e9d575b2592c.png">
> > 
> > 
> > Yes, that's right. I encountered this issue after updated my macOS from Ventura to Sonoma. Please note that I was able to play Dota 2 normally on Ventura.
> 
> it works on windowed mode but you can't select full screen resolution
It does not work

**Comment by Spaider15** [comment]
> > > > same for me, happened after i have installed Mac OS Sonoma <img alt="image" width="295" src="https://user-images.githubusercontent.com/12617760/271366149-26216451-ca51-4feb-a3b9-e9d575b2592c.png">
> > > 
> > > 
> > > Yes, that's right. I encountered this issue after updated my macOS from Ventura to Sonoma. Please note that I was able to play Dota 2 normally on Ventura.
> > 
> > 
> > it works on windowed mode but you can't select full screen resolution
> > It does not work

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/12617760/5de2f90a-aa9b-4ffd-b909-519614999959)

This settings is fine for me

**Comment by Iuriy-Budnikov** [comment]
<img width="269" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/3968373/a0e15243-cabd-418e-a086-b6a7d0fc29f0">

**Comment by viacheslav-boiarchuk** [comment]
Same problem with Sonoma(

**Comment by IharKrasouskiGQ** [comment]
@Gweenteaneko were you able to fix this bug as it was you mentioned? No updates after you tested it

**Comment by L-Ahmadreza** [comment]
Has anyone tested recently to see if the problem is resolved or not?
